raiden-ts/src/transfers/epics.ts

```diff
@@ -852,6 +842,33 @@ export const transferSecretRequestedEpic = (
     }),
   );
 
+const secretReveal = (
```

Comment:
Functions returning observables may have the `$` suffix, for consistency

Reply:
Thanks :) I will change it :+1: